Source/IntegrationTests/TestFiles/LitTests/LitTest/auditor/TestAuditor.dfy-ietf.md.expect

```diff
@@ -10,8 +10,6 @@
 
 * Declaration has explicit `{:axiom}` attribute. SHOULD provide a proof or test.
 
-* Ghost declaration has no body and has an ensures clause. MUST provide a body or add `{:axiom}`.
-
```

Comment:
This seems like something we want to continue reporting in the auditor.

Reply:
But it has `{:axiom}` on it. The comment seemed incorrect.

Reply:
Ah! Got it. I didn't look at the source code it was reporting on. :)